test/fixtures/cmd-src-ref.yml

```diff
@@ -21,8 +21,7 @@ sources:
         mounts:
           - dest: "/foo"
             spec:
-              context:
-                path: .
+              path: .
```

Comment:
For these, we need to not bother setting `path` and instead use an explicit `context: {}`

Reply:
*Not bother setting path when `path` is `.`, `./`,  or `/`